app/views/planning_applications/review/cil_liability/_form.html.erb

```diff
@@ -3,6 +3,8 @@
       class: "govuk-!-margin-top-7" do |form| %>
   <%= form.govuk_error_summary %>
 
+  <%= render "planning_applications/review/cil_liability/audit_trail" %>
+
```

Comment:
you could change line 1 to be

```ruby
<%= form_with model: planning_application,
      url: planning_application_review_cil_liability_url(planning_application),
````
as you pass the reference of planning_application

Reply:
I've removed the unnecessary local assignment instead, since this can simply use the instance variable app/views/planning_applications/review/tasks/_review_assessment.html.erb